### DISCUSSION TITLE
[Feature Request] macOS Text Replacement Integration

### DISCUSSION THREAD

**Discussion by fuzenco** [zed-industries/zed#12641] [Feature Requests]
macOS has Text Replacements at its core. I would love if Zed would recognize these keyboard shortcuts. I do a lot of doc writing (not coding) in Zed and I’d love to use the system shortcuts without reverting to installing a 3rd party text shortcut app.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
